### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by Niputi**
closes https://github.com/Mikibot/sprint-tasks/issues/21

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ENV NODE_ENV=production
+
+COPY package.json yarn.lock ./
+RUN yarn --frozen-lockfile --non-interactive
+
+COPY . .
+RUN yarn build
+
+# Stage 2: Web Server
+FROM nginx:alpine
```
Comment (velddev) [inline Dockerfile]: Are you sure we want to host nginx for every web service on a docker rom?

Comment (Niputi) [reply Dockerfile]: so did https://github.com/Mikibot/web/tree/nginx-refactor

Comment (velddev) [reply Dockerfile]: I remember that was for development usage, but I'm thinking about production usage

Comment (Niputi) [reply Dockerfile]: should i remove it?

Comment (Niputi) [reply Dockerfile]: if I do remove. I'm not sure how I should serve the files

Comment (velddev) [reply Dockerfile]: I'd rather want to think that nginx should be a dockercompose and dockerfile should be only neccessary things

Comment (velddev) [reply Dockerfile]: `docker-compose development`

Comment (Niputi) [reply Dockerfile]: `docker-compose` does not have cmd called `development`

**Pull request merged**